cypress/components/toast/toast.cy.tsx

```diff
@@ -204,6 +207,19 @@ context("Testing Toast component", () => {
           .should("have.css", "justify-content", align);
       }
     );
+    it.each(["top", "bottom"] as ToastProps["alignY"][])(
```

Comment:
Should we have the case for prop "alignY" as centre here too? it.each(["top","centre", "bottom"]